src/gulpfile.babel.js

```diff
@@ -128,19 +131,75 @@ gulp.task( 'styles', () => {
 		.pipe( sourcemaps.write({ includeContent: false }) )
 		.pipe( sourcemaps.init({ loadMaps: true }) )
 		.pipe( autoprefixer( config.BROWSERS_LIST ) )
-		.pipe( sourcemaps.write( './' ) )
+		.pipe( gulpif( ! processOptions.isRTL, sourcemaps.write( './' ) ) ) // Write source map at this step if not RTL.
 		.pipe( lineec() ) // Consistent Line Endings for non UNIX systems.
-		.pipe( gulp.dest( config.styleDestination ) )
```

Comment:
Why did you remove this line? It's required here to create a style file before filtering and processing it further. Sadly, no automated testing in place so, do test and add this line back I suppose. 🙌

Reply:
The line did not removed, just moved to other line. The root problem is steps of styles and stylesRTL task has a bit different pipe order.  I merged pipes from both tasks to support non-RTL and RTL.

![image](https://user-images.githubusercontent.com/695425/46918934-78cccd80-d002-11e8-86ec-bfe1ebd96e1e.png)

Are you ok if i create a separate functions as `processStyle` and `processStyleRTL` for keeping the original code of pipes? I'm new for gulp and i don't known side effects of those pipes.

Reply:
Yes, go for separate two functions. Better than having four.